manifests/repo/yum.pp

```diff
@@ -22,7 +22,7 @@
         baseurl  => "http://yum.puppetlabs.com/el/${::operatingsystemmajrelease}/${::puppet::collection}/${::architecture}",
         enabled  => '1',
         gpgcheck => '1',
-        gpgkey   => 'https://yum.puppetlabs.com/RPM-GPG-KEY-puppetlabs',
+        gpgkey   => 'https://yum.puppetlabs.com/RPM-GPG-KEY-puppet',
```

Comment:
Doh.. I thought GitHub would be smart enough to ignore the other commit.. my bad. 

I'll sync my fork and send another one..